master/custom/factories.py

```diff
@@ -1365,10 +1365,21 @@ def setup(self, **kwargs):
                 env=compile_environ,
             ),
             Compile(
-                name="Compile host libFFI",
+                name="Compile host libffi",
                 command=["python3", "Tools/wasm/emscripten", "make-libffi"],
                 env=compile_environ,
             ),
+        ])
+        if branch != '3.14':
+            # Can enable on 3.14 if/when python/cpython#161845 is merged
```

Comment:
```suggestion
            # Can enable on 3.14 if/when python/cpython#137066 is merged
```
I assume this was meant to point to the backport?